docs/tutorials/tut-minemeld-migration.md

```diff
@@ -0,0 +1,291 @@
+
+<img src="../doc_imgs/tutorials/tut-minemeld-migration/search-office-feed.png"></img>
+
+Let's configure an instance.
+
+<img src="../doc_imgs/tutorials/tut-minemeld-migration/office-feed-configuration-1.png"></img>
+
+As you can see in the screenshot above, Cortex XSOAR provides default values for many of the configuration parameters, as determined by the source of the feed. To configure the integration instance to fetch from the same source as the MineMeld node we are migrating from, we do not need to make any adjustments because the default values for the **Regions** and **Services** configuration parameters are *Worldwide* and *All* respectively, which are the values we need to migrate this particular prototype. 
+
+Now, click `Done` at the bottom right of the configuration window.
+
+Here are two other cases.
+Let's say you want to migrate two other Office 365 prototypes instead of the one previously described, for example,
+```yaml
+  allow_o365-china-exchange:
+    inputs: []
+    output: true
+    prototype: o365-api.china-exchange
+  allow_o365-germany-exchange:
+    inputs: []
+    output: true
+    prototype: o365-api.germany-exchange
+```
+
+We can configure one instance of the _Office 365 Feed_ integration with **Regions** set to *China* and *Germany* and **Services** set to *Exchange*, as shown in the following screenshot.
+
+<img src="../doc_imgs/tutorials/tut-minemeld-migration/office-feed-configuration-2.png"></img>
+
+In the second case, let's say you wanted to migrate the two nodes from your MineMeld configuration that appear as follows,
+```yaml
+  allow_o365-china-exchange:
+    inputs: []
+    output: true
+    prototype: o365-api.china-exchange
+  allow_o365-germany-skype:
+    inputs: []
+    output: true
+    prototype: o365-api.germany-skype
+```
+
+Since these do not overlap, we should configure two instances of the _Office 365 Feed_ integration. One instance with **Regions** set to *China* and **Services** set to *Exchange* and the other instance with **Regions** set to *Germany* and **Services** set to *Skype* as shown in the screenshots below.
+
+<img src="../doc_imgs/tutorials/tut-minemeld-migration/office-feed-configuration-3.png"></img><img src="../doc_imgs/tutorials/tut-minemeld-migration/office-feed-configuration-4.png"></img>
+
+
+## Migrating Output Nodes
+
+You can output indicators with Cortex XSOAR by using two integrations, _Palo Alto Networks PAN-OS EDL Service_ and _Export Indicators Service_. Migrating MineMeld output nodes to Cortex XSOAR is a process that requires looking at the prototype of a given output node, as well as the prototypes of all of the nodes that flow into that output node. We need to do this to understand how to construct the query we will enter when configuring an instance of the _Palo Alto Networks PAN-OS EDL Service_ or _Export Indicators Service_ integration.
```

Comment:
You have some playbooks for that as well, we should mention that

Reply:
i dont think the playbooks are required for the migration